README.md

```diff
@@ -236,3 +236,52 @@ In order to run the tests type the following to your shell:
 ```
 python -m unittest mona_sdk/tests/client_tests.py
 ```
+
+## Asynchronous Client
+
+Same services as the regular client are provided in a non-blocking version in the AsyncClient, in order to give you the option to embed the code you write using this package with asyncIO code.
+
+Everything described for the regular client applies also to the AsyncClient, with these additions:
+
+- **Methods names:**
+  - Each method of the AsyncClient in its async form gets an "_async" suffix (for example, export turns to export_async, export_batch turns to export_batch_async, etc.) 
+- **Client parameters:** 
+  - event_loop (_UnixSelctorEventLoop): (optional) The event loop that will manage the threads. 
+  - executor (TreadPoolExecutor): (optional) The executor the will manage the thread pool. 
+  
+  
+- **Methods keyword parameters:**
```

Comment:
I think this section should appear above, right after the text I suggested. You should mention that if supplied in the client parameters (this means the constructor? please be explicit) then these params will be used and don't have to be passed here (or can be overridden). Then you should add these params into the description of all the params for the constructor and mention there that it's relevant to the async version with a link to below or something...